pkg/services/sqlstore/migrations/annotation_mig.go

```diff
@@ -123,7 +123,21 @@ func addAnnotationMig(mg *Migrator) {
 	mg.AddMigration("Make epoch_end the same as epoch", NewRawSqlMigration("UPDATE annotation SET epoch_end = epoch"))
 	mg.AddMigration("Move region to single row", &AddMakeRegionSingleRowMigration{})
 
-	// TODO! drop region_id column?
+	mg.AddMigration("Remove index org_id_epoch on annotation table", NewDropIndexMigration(table, &Index{
+		Cols: []string{"org_id", "epoch"}, Type: IndexType,
+	}))
+
+	mg.AddMigration("Remove index org_id_dashboard_id_panel_id_epoch on annotation table", NewDropIndexMigration(table, &Index{
+		Cols: []string{"org_id", "dashboard_id", "panel_id", "epoch"}, Type: IndexType,
+	}))
+
+	mg.AddMigration("Add index for org_id_dashboard_id_epoch_epoch_end on annotation table", NewAddIndexMigration(table, &Index{
+		Cols: []string{"org_id", "dashboard_id", "epoch", "epoch_end"}, Type: IndexType,
+	}))
+
+	mg.AddMigration("Add index for org_id_dashboard_id_panel_id on annotation table", NewAddIndexMigration(table, &Index{
+		Cols: []string{"org_id", "dashboard_id", "panel_id"}, Type: IndexType,
+	}))
```

Comment:
Why do we need this index?  It would only ever be used when querying annotations for a specific panel with no time range, as soon as you have a time range the index above is preferable.

Reply:
This use case is for [alert state history](https://github.com/grafana/grafana/blob/master/public/app/features/alerting/StateHistory.tsx#L27) filtering on org id, dashboard and panel id.

Reply:
Ok